libretro/libretro.cpp

```diff
@@ -198,7 +200,16 @@ static void update_variables(void)
    bool geometry_update = false;
    char key[256];
    struct retro_variable var;
-   var.key = "snes9x_overclock";
+
+   var.key = "snes9x_hires_blend";
+   var.value = NULL;
+
+   if (environ_cb(RETRO_ENVIRONMENT_GET_VARIABLE, &var))
+      hires_blend = !strcmp(var.value, "disabled") ? false : true;
+   else
+      hires_blend = false;
+
+   var.key = "snes9x_overclock_superfx";
```

Comment:
I think this needs to match the name above. Which one is used in your core? With _superfx added?

Reply:
ahh true, between patches I missed that one, I'll fix it tomorrow